tests/system/test_integration.py

```diff
@@ -69,6 +69,11 @@ def test_load_docs_with_template_and_add_transaction(self):
         # compare existing ES documents for transactions with new ones
         rs = self.es.search(index=index_transaction)
```

Comment:
looks like we can change these to `wait_for_events` and consolidate the `event.ingested` munging

Reply:
@simitt I made some changes after your review to get tests passing - we now disregard the value in `event.ingested` in all approval tests - would peek at b3778dc9422d66443426995f654144d1d3dd866e and ensure you still approve?